src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -4009,6 +4037,10 @@ int wm_vuldet_oval_xml_parser(OS_XML *xml, XML_NODE node, wm_vuldet_db *parsed_o
     static const char *XML_CRITERION = "criterion";
     static const char *XML_TEST_REF = "test_ref";
     static const char *XML_COMMENT = "comment";
+    static const char *XML_HREF = "href";
+    static const char *XML_IMPACT = "impact";
+    static const char *XML_CWE = "cwe";
+    static const char *XML_CVSS3 = "cvss3";
```

Comment:
Let's remove these new headers (and their uses), in order to avoid getting this information from the OVAL, as we fill these variables with the NVD information.